ReactWindows/ReactNative.Shared/Views/Text/FontStyleHelpers.cs

```diff
@@ -99,5 +101,87 @@ private static FontWeight ConvertNumeric(int fontWeightNumeric)
             }
 #endif
         }
+
+#if !WINDOWS_UWP
+        public static FontStyle? ConvertFontStyle(string fontStyleValue)
+        {
+            FontStyle? fontStyle = null;
+
+            if (!fontStyleValue.IsNullOrEmpty())
```

Comment:
This was a coding error that seems to work with debug, but not release modes... Changed this to the correct implementation string.IsNullOrEmpty